api/src/main/kotlin/edu/wgu/osmt/collection/CollectionEsRepo.kt

```diff
@@ -34,6 +36,10 @@ interface CustomCollectionQueries : FindsAllByPublishStatus<CollectionDoc> {
             Sort.by("name.keyword").descending()
         )
     ): SearchHits<CollectionDoc>
+
+    fun deleteIndex() {
+        elasticSearchTemplate.indexOps(IndexCoordinates.of(INDEX_COLLECTION_DOC)).delete();
```

Comment:
we can skip semicolon here.

Reply:
Yup, good catch. I copy/pasted in from Java.